test/run_tests.sh

```diff
@@ -226,6 +226,8 @@ function run_xla_op_tests3 {
   run_test "$CDIR/spmd/test_xla_distributed_checkpoint.py"
   run_test "$CDIR/spmd/test_xla_spmd_python_api_interaction.py"
   run_test "$CDIR/spmd/test_dtensor_integration.py"
+  run_test "$CDIR/spmd/test_dtensor_integration2.py"
```

Comment:
do we need this on TPU CI  as well or it is ok to leave out?

Reply:
Ohhh i think it's ok to leave out. Want to run this sanity check on TPU!